packages/manager-components/src/components/templates/onboarding/onboarding.component.tsx

```diff
@@ -108,7 +108,7 @@ export const OnboardingLayout: React.FC<OnboardingLayoutProps> = ({
       {!hideHeadingSection && (
         <section className="flex flex-col items-center">
           {(img?.src || placeholderSrc) && (
-            <div className="flex justify-center pt-8 max-h-28">
+            <div className="flex justify-center pt-8 max-h-28 w-full">
```

Comment:
Its new spec by the design team ?

Reply:
The top image does not display at all if there is not "width: 100%"
I think it's because of the svg format. For png format it will still work the same.

Reply:
👍